shared/src/test/diff/mlscript/Ascribe.mls

```diff
@@ -1,61 +1,71 @@
 
 "hello": 'a
 //│ res: "hello"
+//│    = 'hello'
 
 :e
 "hello": 'a -> 'a
 //│ ╔══[ERROR] Type mismatch in type ascription:
-//│ ║  l.6: 	"hello": 'a -> 'a
+//│ ║  l.7: 	"hello": 'a -> 'a
 //│ ║       	^^^^^^^
 //│ ╟── expression of type `"hello"` is not a function
 //│ ╟── Note: constraint arises from function type:
-//│ ║  l.6: 	"hello": 'a -> 'a
+//│ ║  l.7: 	"hello": 'a -> 'a
 //│ ╙──     	         ^^^^^^^^
 //│ res: 'a -> ('a | error)
 
 { name = "Bob" } : { name: 'a }
 //│ res: {name: "Bob"}
+//│    = 'Bob'
```

Comment:
Interesting. This should evaluate to a record, not just a string.

Reply:
Generated code is `{  name: "Bob"  };`. So I should wrap object literals with parentheses otherwise the parser interprets them as blocks with a label followed by a string literal.

Reply:
Wow, JavaScript's parser never ceases to amaze me 😂

Reply:
By the way, I believe some generated code trigger bad [semicolon insertion](https://stackoverflow.com/questions/2846283/what-are-the-rules-for-javascripts-automatic-semicolon-insertion-asi) position somewhere...